src/SearchResults.js

```diff
@@ -0,0 +1,41 @@
+import moment from 'moment';
+import SearchRow from './SearchRow';
+
+function SearchResults(props) {
+  
+  let tr = props.results.map((element) => (
+    <SearchRow
+       key = {element.id}
+       id = {element.id}
```

Comment:
I haven't seen props passed quite like this before, it'd generally better ot just do
`id={element.id}` over `id = {element.id}`

Reply:
yes, I fixed it